altair/api.py

```diff
@@ -124,7 +124,7 @@ class SortItems(BaseObject):
 
     name = T.Unicode(default_value=None, allow_none=True)
     aggregate = T.Enum(['avg', 'sum', 'min', 'max', 'count'],
```

Comment:
FYI, in vega-lite 0.8, `avg` will be changed to `mean` for consistency with Datalib.  We will also support a bunch of other Vega/Datalib. aggregation operator.  

Vega-lite also include a [JSON-schema](https://github.com/vega/vega-lite/blob/gh-pages/spec.json) in our [releases](https://github.com/vega/vega-lite/releases).  You might want to consider referring to the JSON-schema rather than hard-coding supported enum.  That said, hard-coding maybe better if you decide to support only a subset of values.